### ISSUE TITLE
SD card is readable under Breath but cannot boot breath from SD card

### ISSUE THREAD

**Issue by RodBarnes** [cb-linux/breath#124]
Not sure if this is expected behavior or what I would check to see why it cannot boot from the SD card.

_Why does this matter?_  Not a big deal but would be nice.  My device -- as do, I think, many of the Chromebooks -- have a micro SD card slot (or at least a standard SD slot).  Being able to insert a card there and not have a big USB dongle sticking out would be a plus.  Also, it would be possible to run it on the SD card for a while without having to install to internal storage while we're still doing a lot of development.

**Comment by MilkyDeveloper** [comment]
Some devices - mine included - support booting from an SD Card. I suspect that this is a driver-related issue. What’s the output of lsusb and lspci -vvvv within ChromeOS?

**Comment by RodBarnes** [comment]
Under ChromeOS:
```
chronos@localhost / $ lsusb
Bus 002 Device 002: ID 0951:1666 Kingston Technology DataTraveler G4
Bus 002 Device 001: ID 1d6b:0003 Linux Foundation 3.0 root hub
Bus 001 Device 003: ID 8087:0a2a Intel Corp. 
Bus 001 Device 002: ID 04f2:b567 Chicony Electronics Co., Ltd HD WebCam
Bus 001 Device 001: ID 1d6b:0002 Linux Foundation 2.0 root hub
chronos@localhost / $ lspci -vvvv
bash: lspci: command not found
```

**Comment by RodBarnes** [comment]
@MilkyDeveloper Any thoughts on how I can fix this?  (I know your focused on audio -- and grateful for that.  But if this is a simple fix, it would really help my testing capability as I have way more micro SD cards than I do USB flash drives.  Thanks.)

**Comment by RodBarnes** [comment]
@MilkyDeveloper  Another thought:  Since I can successfully boot from the SD when using LEGACY_BOOT, it seems this is related to the coreboot/uboot used on the Chromebook.  If you have resources on this topic, please direct me.  Meanwhile, I'll continue to investigate.

**Comment by hede5562** [comment]
I'm using a modified kernel.flags for my system as the SD-Card is no /dev/sdX device but /dev/mmcblk0: 
https://github.com/hede5562/breath/blob/main/kernel.modern.flags
`console=tty1 root=/dev/mmcblk0p2 i915.modeset=1 rootwait rw fbcon=logo-pos:center,logo-count:1 loglevel=0 splash`

**Comment by MilkyDeveloper** [comment]
I'll probably just use UUIDs in the future.

**Comment by hede5562** [comment]
Good idea. The UUID is fixed after creating the partition table and filesystem. It should be possible to find them out while creating the image or usb media.
